server/spinwick.go

```diff
@@ -88,15 +105,86 @@ func (s *Server) handleCreateSpinWick(pr *model.PullRequest, size string, withLi
 
 }
 
-func (s *Server) createKubeSpinWick(pr *model.PullRequest) *spinwick.Request {
+// createCloudSpinwickWithCWS will use the defined CWSCloudInstance to create a new user/customer and
+// instantiate a new MM cloud installation
+func (s *Server) createCloudSpinWickWithCWS(pr *model.PullRequest, size string) *spinwick.Request {
 	request := &spinwick.Request{
 		InstallationID: "n/a",
 		Error:          nil,
 		ReportError:    false,
 		Aborted:        false,
 	}
 
-	logger := log.WithField("PR", pr.RepoName+": #"+string(pr.Number))
+	uniqueID := s.makeSpinWickID(pr.RepoName, pr.Number)
+	spinwickURL := fmt.Sprintf("https://%s.%s", uniqueID, s.Config.DNSNameTestServer)
+	username := fmt.Sprintf("user-%s@example.mattermost.com", uniqueID)
+	password := "Cws@User123"
```

Comment:
We probably want these to be environment variables. Matterwick is open source and we probably don't want these credentials publicly viewable, since CWS is private

Reply:
Yep but we're going to share those credentials as a comment in the PR in github for the server/webapp projects so it'll be public anyway. I can change the password but those are for the recent created user that will be destroyed

Reply:
i prefer as a variable as well

Reply:
Ok, changing to environment variable

Reply:
What do you think about using a random programmatic password instead of having the same for everyone? That way we don't need variables and the password is for just one usage

Reply:
Change done